numpy/core/fromnumeric.pyi

```diff
@@ -248,13 +248,15 @@ def all(
     axis: None = ...,
     out: None = ...,
     keepdims: Literal[False] = ...,
+    where: _ArrayLikeBool_co = ...,
```

Comment:
```suggestion
    *,
    where: _ArrayLikeBool_co = ...,
```
As `where` consistently seems to be keyword-only. Note that the same comment applies to all functions here.